journals/apps/journals/management/commands/publish_journals.py

```diff
@@ -66,18 +68,28 @@ def _handle_create(self, options):
                 journal.organization.site.siteconfiguration.discovery_journal_api_client,
                 journal_meta_data.get_discovery_data()
             )
-        except HttpClientError as err:
+        except (HttpClientError, HttpServerError) as err:
             # TODO - roll back journal updates if this fails
-            raise CommandError('Error publishing to discovery-service: {}'.format(err.content))
+            err_str = 'Error publishing to discovery-service, discovery data={discovery_data} err={err}'.format(
+                discovery_data=journal_meta_data.get_discovery_data(),
```

Comment:
Might make sense to `get_discovery_data()` once before the try/except and use it in both instances. It's probably not a performance hit currently, but we can verify the data is the same that way.